packages/venia-concept/.graphqlconfig

```diff
@@ -0,0 +1,16 @@
+{
+    "projects": {
+        "venia": {
+            "schemaPath": "lastCachedGraphQLSchema.json",
+            "extensions": {
+                "endpoints": {
+                    "default": "${env:MAGENTO_BACKEND_URL}graphql"
```

Comment:
Oops! This should have had a `/`.

[![Image from Gyazo](https://i.gyazo.com/3f5e1550a20c1dc605cef50f8f6e873f.png)](https://gyazo.com/3f5e1550a20c1dc605cef50f8f6e873f)